boards/samr30-xpro/include/periph_conf.h

```diff
@@ -34,14 +34,23 @@ extern "C" {
  * @name    Timer peripheral configuration
  * @{
  */
-#define TIMER_NUMOF        (1U)
-#define TIMER_0_EN         1
+static const tc32_conf_t timer_config[] = {
+    {   /* Timer 0 - System Clock */
+        .dev            = TC0,
+        .flags          = TC_CTRLA_MODE_COUNT32,
+        .irq            = TC0_IRQn,
+        .mclk           = &MCLK->APBCMASK.reg,
+        .mclk_mask      = MCLK_APBCMASK_TC0 | MCLK_APBCMASK_TC1,
+        .gclk_id        = TC0_GCLK_ID,
+        .gclk_src       = GCLK_PCHCTRL_GEN(0),
+        .prescaler      = TC_CTRLA_PRESCALER(4),
+    }
+};
 
 /* Timer 0 configuration */
-#define TIMER_0_DEV        TC0->COUNT32
-#define TIMER_0_CHANNELS   1
-#define TIMER_0_MAX_VALUE  (0xffffffff)
-#define TIMER_0_ISR        isr_tc0
+#define TIMER_0_CHANNELS    2
```

Comment:
Should be 1?

Reply:
samr30 also supports two compare/capture channels in 16/32bit mode.